docs/manual/manual.md

```diff
@@ -0,0 +1,294 @@
+There are two ways to get a haxophone:
+
+1. The fastest way: switch over to crowdsupply and order it there. You need to buy some more stuff as well: a Raspberry Pi Zero (or a Pi 3 or a Pi 4) and an SD card with 16 GB (see [https://www.crowdsupply.com/cardona-bits/haxophone](https://www.crowdsupply.com/cardona-bits/haxophone))
+
+2. You can build your own! The bits and pieces needed to build the haxophone and how to assemble it can be found here: [Assembly Instructions](https://github.com/cardonabits/haxo-hw/blob/main/docs/assembly.md)
+
+ <a name="howto-use-this-manual"></a>
+
+### How to Use This Manual
+
+If you are using your haxophone for the first time go to [Unpacking and Setting Up](#unpackung-and-setting-up).
+
+If you are a musician and you want to get started playing quickly: Got to section [Playing](#playing).
+
+if you are a hacker-hobby-musician or a musician-hobby-hacker then go to [Hacking](#hacking).
+ 
+If you are an advanced hacker go to [Hacking](#hacking)
+
+ <a name="using-the-haxophone"></a>
+
+## Using the Haxophone
+ <a name="unpackung-and-setting-up"></a>
+
+### Unpacking and Setting Up
+When you receive your haxophone from crowdsupply it comes with an instruction, some washi tape and screws.
+
+![](./images/haxophone_unpacked.jpg)
+
+To get started, you will need the following additional items:
+
+1. Raspberry Pi Zero or Zero W or Pi 3 or Pi 4.
+2. PC or notebook with SD card reader
+3. SD card with 16 or 32 GB memory
+4. USB power supply with a micro USB-B plug 
+5. Headphones, earphones or active speaker
+6. Optional: a saxophone mouthpiece
+7. Optional: HDMI cable with HDMI mini connector (not micro)
+8. Optional: USB keyboard
+9. Optional: USB cable, USB network adapter or USB hub with network adapter
+
+![](./images/items_needed_for_getting_started.jpg)
+
+ <a name="connect-and-install-software"></a>
+
+#### Connect and Install the Software
+1. **Connect the Raspberry Pi with the Haxophone:**
+   Put the Raspberry Pi 40-pin connector plug on the haxophone 40-pin connector port. Take care that the USB power supply port on the Raspberry Pi faces the bottom of the haxophone, aligning with the mini-jack connector direction. 
+
+![](./images/connect_raspberry_pi_to_haxophone.jpg)
+
+2. **Connect a Headphone to the Haxophone**
+Connect a headphone, earphone or active speaker to your haxophone using the mini-jack on the bottom edge
+
+3. **Optional: Attach a Saxophone Mouthpiece:**
+   If you want you can attach a saxophone mouthpiece to the haxophone's neck. Use masking, electrical or washi tape to adjust the neck's thickness to match your mouthpiece.
+
+![](./images/with_mouthpiece.png)
+
+4. **Optional: Attach a Neck Strap:**
+You can attach a neck strap using the eyelet just on top of the thumb rest.
+
+5. **Install the Software on the SD Card**
+
+5a) Download the pre-compiled software image from one the following links:
+
+* Raspberry Pi Zero and Zero W: [https://bit.ly/haxo-rpiz](https://bit.ly/haxo-rpiz)
```

Comment:
* Raspberry Pi Zero 2: [https://bit.ly/haxo-rpiz2](https://bit.ly/haxo-rpiz2)

Reply:
I added this link in the next version